src/zenml/cli/base.py

```diff
@@ -230,13 +230,18 @@ def go() -> None:
             f"{zenml_tutorial_path} already exists! Continuing without cloning."
         )
 
-    ipynb_files = [
-        fi for fi in os.listdir(zenml_tutorial_path) if fi.endswith(".ipynb")
-    ]
+    # get list of all .ipynb files in zenml_tutorial_path
+    ipynb_files = []
+    for dirpath, _, filenames in os.walk(zenml_tutorial_path):
```

Comment:
This will print the full path now so should be fine:

<img width="577" alt="Screenshot 2022-05-24 at 16 11 18" src="https://user-images.githubusercontent.com/23118541/170056625-fe24423b-3b1f-4e3e-93d1-721fd10f204b.png">

Reply:
Alright!